FiveCalls/FiveCalls/ContactCircle.swift

```diff
@@ -49,12 +49,23 @@ struct ContactCircle: View {
                 Circle()
             }
     }
+    
+    func issueCalledOn(issueID: Int, contactID: String) -> Bool {
```

Comment:
@cselin you'll probably notice this is duplicated in two places... where would it make sense to share logic like this? It needs issue info, contact info and app state info!

Reply:
Oh yeah, state extension sounds great, will do that for now.